analyzer-comments/java/annalyns-infiltration/avoid_unnecessary_parenthesis.md

```diff
@@ -0,0 +1,13 @@
+# Some parenthesis are redundant
```

Comment:
I believe @SleeplessByte meant for you to rename the file to `some_parentheses_are_redundant.md`, and then updating the header to match the file name 🙂

Reply:
Fair enough, it makes more sense after all. I also made the changes to java-analyzer for better similarity between projects